pkgs/development/python-modules/affine/default.nix

```diff
@@ -0,0 +1,21 @@
+{ buildPythonPackage, pytest, lib, fetchPypi }:
+
+buildPythonPackage rec {
+  pname = "affine";
+  version = "2.2.1";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "0j3mvcnmgjvvm0znqyf7xylq7i89zjf4dq0g8280xs6bwbl5cvih";
+  };
+
+  checkInputs = [ pytest ];
```

Comment:
You might want to add
```
checkPhase = ''
  py.test
'';
```
because that runs 62 instead of only 46 tests and it's what's upstream uses.

Reply:
Sure, will do that